_data/authors.yml

```diff
@@ -310,6 +316,12 @@
   description: is a Product Owner at ePages. She started her job career as an Editor where she learned to love user-centric values. Taking these values over to Product Management truly matters to her.
   image: kdeharde.jpg
 
+- id: kkoessler
+  name: Katharina
+  full_name: Katharina Kössler
+  description: is HR Manager at ePages. She loves to spread good humour and motivation, and find new colleagues for the teams at ePages.
```

Comment:
I'd suggest some minor changes, e.g. avoid two times ePages. "is an HR Manager at ePages. She loves to spread good humor and motivation, and to find new colleagues for our different teams."